ospd/parser.py

```diff
@@ -157,6 +159,21 @@ def __init__(self, description: str) -> None:
             action='store_true',
             help='Display all protocol commands',
         )
+        parser.add_argument(
+            '--max-scans',
+            default=DEFAULT_MAX_SCAN,
+            type=int,
+            help='Max. amount of parallel task that can be started. '
+            'Default %(default)s, disabled',
+        )
+        parser.add_argument(
+            '--check-free-memory',
+            default=False,
+            type=self.str2bool,
```

Comment:
What's the reason fo not using `action='store_true'` here?

Reply:
The action='store_true' is not compatible with the 'type' argument. In this case the default action='store' is taken.

Reply:
I mean why not using just `ospd-openvas --check-free-memory` ? That's what `action='store_true'` is about  :-)

Reply:
not always will be used the command line option, but can be set in the ospd.conf config file.
In this way it accepts for True one of 'yes', 'true', 't', 'y', '1' , otherwise is set to False. I think in this way it is more flexible for the config file case.

Reply:
not always will be used the command line option, but can be set in the ospd.conf config file.
In this way it accepts for True one of 'yes', 'true', 't', 'y', '1' , otherwise is set to False. I think in this way it is more flexible for the config file case.